programs/locked-voter/src/instructions/withdraw_partial_unstaking.rs

```diff
@@ -0,0 +1,112 @@
+use crate::*;
+use anchor_spl::token;
+
+/// Accounts for [voter::withdraw_partial_unstaking].
+#[derive(Accounts)]
+pub struct WithdrawPartialUnstaking<'info> {
+    /// The [Locker] being exited from.
+    #[account(mut)]
+    pub locker: Box<Account<'info, Locker>>,
+
+    /// The [Escrow] that is being closed.
+    #[account(mut, has_one = locker, has_one= owner)]
+    pub escrow: Box<Account<'info, Escrow>>,
+
+    /// The [Escrow] that is being closed.
+    #[account(mut, has_one = escrow, close = payer)]
+    pub partial_unstake: Box<Account<'info, PartialUnstaking>>,
+
+    /// Authority of the [Escrow].
+    pub owner: Signer<'info>,
+
+    /// Tokens locked up in the [Escrow].
+    #[account(mut, constraint = escrow.tokens == escrow_tokens.key())]
+    pub escrow_tokens: Account<'info, TokenAccount>,
```

Comment:
I think the escrow_tokens account might leave dangling when
1. increase_lock_amount 100
2. open_partial_unstaking 100
3. when lock release, withdraw_partial_unstaking 100
4. escrow_tokens leave dangling, they can't close it using withdraw too.